doc/classes/Dictionary.xml

```diff
@@ -21,11 +21,19 @@
 		var points_dict = {"White": 50, "Yellow": 75, "Orange": 100}
 
 		# Alternative Lua-style syntax.
+		#
 		# Doesn't require quotes around keys, but only string constants can be used as key names.
-		# Additionally, key names must start with a letter or an underscore.
+		# Additionally, for key names to be usable without quotes, they must start with a letter
+		# or an underscore and must not be reserved identifers (such as `class` or `while`).
 		# Here, `some_key` is a string literal, not a variable!
```

Comment:
Should this be changed?
> Here, `some_key` is a string literal

If it was true, `{class = "xyz"}`, `class` would be a string literal, but parser don't understand it.

Reply:
`some_key` is understood as a string literal, so I'd say it's still correct with this new example. I've moved the comment further below to be closer to the dictionary declaration.